CMakeLists.txt

```diff
@@ -235,18 +278,25 @@ if (MSVC)
   target_include_directories(OpenSCAD SYSTEM PRIVATE ${HARFBUZZ_INCLUDE_DIRS}/harfbuzz)
   list(APPEND COMMON_LIBRARIES harfbuzz::harfbuzz)
 
-  find_package(unofficial-fontconfig CONFIG REQUIRED)
-  list(APPEND COMMON_LIBRARIES unofficial::fontconfig::fontconfig)
+  find_package(fontconfig REQUIRED)
+  list(APPEND COMMON_LIBRARIES ${VCPKG_LIBRARY_DIR}/fontconfig.lib)
```

Comment:
The whole point of `find_package` is that it sets [various variables](https://cmake.org/cmake/help/latest/manual/cmake-developer.7.html#standard-variable-names) eg `[packageName]_LIBRARIES`, so that you don't have to construct their paths yourself.  

In this case vcpkg was previously providing a package with unofficial in the name, but it seems they must have dropped that recently.  Still there should be either standard variable name set, or a namespaced target (with double colons `::` like in the removed code)